src/scala/annotation/static.scala

```diff
@@ -0,0 +1,13 @@
+package scala.annotation
+
+import scala.annotation.meta._
+
+/** https://github.com/scala/scala.github.com/pull/491 */
+
+@field
+@getter
+@beanGetter
+@beanSetter
+@param
```

Comment:
For consistency. Now if _any_ kind of member is created from a `@static` one it has to still be `@static`